app/assets/javascripts/admin/addon/controllers/admin-emojis-new.js

```diff
@@ -0,0 +1,27 @@
+import Controller from "@ember/controller";
+import EmberObject, { action, computed } from "@ember/object";
+import { service } from "@ember/service";
+
+const ALL_FILTER = "all";
+
+export default class AdminEmojisNewController extends Controller {
+  @service router;
+
+  @computed("model")
+  get emojiGroups() {
+    return this.model.mapBy("group").uniq();
+  }
+
+  @computed("emojiGroups.[]")
+  get sortingGroups() {
+    return [ALL_FILTER].concat(this.emojiGroups);
+  }
```

Comment:
These two are duplicated between the "index" and the "new" controller. What'd be the canonical way to share this?

Reply:
This duplication doesnt feel too bad to me